liquidprompt

```diff
@@ -1419,10 +1421,16 @@ _lp_runtime()
         (( _LP_RUNTIME_SECONDS >= 60 )) && echo -n $((_LP_RUNTIME_SECONDS % 3600 / 60))m
         echo -n $((_LP_RUNTIME_SECONDS % 60))"s${NO_COL}"
     fi
+    if (( LP_ENABLE_RUNTIME_BELL && _LP_RUNTIME_SECONDS >= LP_RUNTIME_BELL_THRESHOLD ))
+    then
+        #bash reprints $PS1 on every SIGWINCH, so we tput to stderr here so the bell is not
+        #captured in PS1.  This way the bell only rings once, not on every terminal resize.
+        tput bel >&2
```

Comment:
I just learned this this week myself: note that `tput` isn't exactly universal in its capability codes. Stupid POSIX doesn't actually define what or how `tput` should define the codes that the user uses.

Some BSDs use slightly different codes. We actually already code for this here:
https://github.com/nojhan/liquidprompt/blob/da9e97f8025f01fbe4c0ff06eccdc1454b63256e/liquidprompt#L226-L237
Reading the [FreeBSD docs](https://www.freebsd.org/cgi/man.cgi?query=terminfo&sektion=5&apropos=0&manpath=FreeBSD+12.1-RELEASE+and+Ports), it looks like the code of `bl` should work:
```suggestion
        { tput bel || tput bl ; } 2>/dev/null >&2
```
Note the addition of `2>/dev/null`. If the `tput` command errors out for some reason (or on the first try), I don't want it to be printed to terminal.

I don't have any BSD environments to test on (hoping to fix that soon), but I'm comfortable with this change since it matches our other uses of `tput`.